src/java.base/share/classes/java/lang/Class.java

```diff
@@ -236,13 +236,14 @@ private static void runtimeSetup() {
      * This constructor is not used and prevents the default constructor being
      * generated.
      */
-    private Class(ClassLoader loader, Class<?> arrayComponentType, int mods, ProtectionDomain pd) {
+    private Class(ClassLoader loader, Class<?> arrayComponentType, char mods, ProtectionDomain pd, boolean isPrim) {
         // Initialize final field for classLoader.  The initialization value of non-null
         // prevents future JIT optimizations from assuming this final field is null.
```

Comment:
To add a bit more depth to this comment, I'd add.

"The following assignments are done directly by the VM without calling this constructor."
Or something to that effect.

Reply:
Okay, that's a good comment.  I'll add it.